File: Progress/Pertemuan 5/TugasPertemuan5.java
import java.util.Scanner;

public class TugasPertemuan5 {
    public static void main(String[] args) {
        System.out.println("Pendataan IPK mahasiswa semester terakhir");
        System.out.println(" ");
        
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.print("Masukan nama lengkap anda = ");
            String namaLengkap = scanner.nextLine();
            System.out.print("Masukan nilai IPK anda = ");
            Double ipk = scanner.nextDouble();
            System.out.print("Masukan lama masa studi anda = ");
            Double tahun = scanner.nextDouble();

            System.out.println(" ");

            if (ipk >= 3.51 && tahun <=4) {
                System.out.println("Selamat " + namaLengkap + " dinyatakan lulus Summa Cumlaude");
            } else if (ipk >= 3.51 ) {
                System.out.println("Selamat " + namaLengkap + " dinyatakan lulus Cumlaude");
            } else if (ipk >= 3.01 && ipk < 3.51) {
                System.out.println("Selamat " + namaLengkap + " dinyatakan lulus dengan sangat memuaskan");
            } else if (ipk >= 2.76 && ipk < 3.01) {
                System.out.println("Selamat " + namaLengkap + " dinyatakan lulus memuaskan");
            } else if (ipk >= 2.00 && ipk < 2.76) {
                System.out.println("Selamat " + namaLengkap + " dinyatakan lulus dengan cukup");
            } else if (ipk < 2.00) {
                System.out.println("Maaf " + namaLengkap + " dinyatakan tidak lulus");
            } else if (ipk > 4.00 && ipk < 0) {
                System.out.println("Data yang dimasukan tidak valid");
            }
        }
        System.out.println("             Terima Kasih");
    }             
}

File: Progress/Pertemuan 5/TugasPertemuan5.py
print();
print("Biodata Diri."            );
print();
print("Masukkan data anda dengan benar!");
print();
nDepan      = input("Nama Depan     : ");
nBelakang   = input("Nama Belakang  : ");
nim         = input("NIM            : ");
pStudi      = input("Program Studi  : ");
fakultas    = input("Fakultas       : ");
thnMasuk    = input("Tahun Masuk    : ");
thnLulus    = input("Tahun Lulus    : ");
ipk         = input("IPK            : ");
print();

lamaStudi = int(thnLulus) - int(thnMasuk) + 1

print("          Kartu Hasil Studi:           ");
print("Nama Lengkap    : " + nDepan + " " + nBelakang);
print("NIM             : " + nim);
print("Program Studi   : " + pStudi);
print("Fakultas        : " + fakultas);
print("IPK             : " + ipk);
print("Lama Studi      : " + str(lamaStudi) + " tahun")

#program if, else, elif
if (float(ipk) > 4.00 or float(ipk) < 0) :
    print("Predikat Lulus  : Tidak Valid");
    print("======================================");
elif (float(ipk) >= 3.51 and int(lamaStudi) <= 4) :
    print("Predikat Lulus  : Dengan Pujian Tertinggi (Summa Cumlaude)");
    print("======================================");
elif (float(ipk) >= 3.51) :
    print("Predikat Lulus  : Dengan Pujian (Cumlaude)");
    print("======================================");
elif (float(ipk) >= 3.01 and float(ipk) < 3.51) :
    print("Predikat Lulus  : Sangat Memuaskan");
    print("======================================");
elif (float(ipk) >= 2.76 and float(ipk) < 3.01) :
    print("Predikat Lulus  : Memuaskan");
    print("======================================");
elif (float(ipk) >= 2.00 and float(ipk) < 2.76) :
    print("Predikat Lulus  : Cukup");
    print("======================================");
elif (float(ipk) < 2.00) :
    print("Predikat Lulus  : Tidak Lulus");
    print("======================================");
